step-templates/terraform-custom-script.json

```
@@ -0,0 +1,109 @@
+{
+    "Id": "1d2a62d0-1f12-4417-8566-4ca1b8e6a69c",
+    "Name": "Execute Custom Terraform Script with Package",
+    "Description": "Run a custom terraform script using a package and an Azure Account.\n\nE.g. run \"terraform taint some-resource\" in the context of your terraform package files and authenticated against Azure using credentials in an Octopus Azure Account variable. This step operates very similarly to the built-in Terraform Apply step but allowing you the control to specift custom terraform commands. Very useful for runbooks and terraform import.",
```

There is a typo here in 'specift'. 
```suggestion
    "Description": "Run a custom terraform script using a package and an Azure Account.\n\nE.g. run \"terraform taint some-resource\" in the context of your terraform package files and authenticated against Azure using credentials in an Octopus Azure Account variable. This step operates very similarly to the built-in Terraform Apply step but allowing you the control to specify custom terraform commands. Very useful for runbooks and terraform import.",
```